Cargo.toml

```diff
@@ -1,15 +1,14 @@
 [package]
 name = "ssb-validate2-rsjs-wasm"
-version = "0.4.0"
+version = "0.5.0"
 authors = ["Andrew Reid <glyph@mycelial.technology>"]
 edition = "2018"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
+ssb-validate = { git = "https://github.com/sunrise-choir/ssb-validate" }
+ssb-verify-signatures = { git = "https://github.com/sunrise-choir/ssb-verify-signatures" }
```

Comment:
Just a comment, not a request: what's the situation with publishing these to crates.io?

Reply:
Good question! I've been meaning to check-in with Mikey about publishing the updates. I think now is a good time to do that since we've more-or-less reached stability with our requirements.